src/extension/features/accounts/hide-memos/index.css

```diff
@@ -0,0 +1,11 @@
+body.toolkit-hide-memos .ynab-grid-cell-memo {
+  display: none !important;
+}
+
+@-moz-document url-prefix() {
+  body.toolkit-hide-memos .ynab-grid-cell-memo {
+    visibility: collapse !important;
```

Comment:
I know I said one would expect `visibility: collapse` to work, and I believe it was when I wrote it. However, after testing #2119, I am seeing that it doesn't, but `visibility: hidden` does.

Was `visibility: collapse` working for anyone else on Firefox?

Reply:
i was using hidden before! 😝 Is it breaking for you?  Imgs?